src/components/Helpers/dictionaries.ts

```diff
@@ -0,0 +1,5 @@
+export const personRoleMap = {
+  "PERSON_ROLE_OWNER": "Eigenaar",
+  "PERSON_ROLE_RESIDENT": "Bewoner",
+  "PERSON_ROLE_MIDDLEMAN": "Tussenpersoon"
+}
```

Comment:
Would have added the `PersonRole` type here. Like: `export type PersonRole = keyof typeof personRoleMap`. And remove the export everywhere else.